apollo-router/src/plugins/expose_query_plan.rs

```diff
@@ -66,10 +66,11 @@ impl Plugin for ExposeQueryPlan {
         &self,
         service: BoxService<RouterRequest, RouterResponse, BoxError>,
     ) -> BoxService<RouterRequest, RouterResponse, BoxError> {
-        let conf_enabled = self.enabled;
+        let conf_enabled =
+            self.enabled || std::env::var(ENABLE_EXPOSE_QUERY_PLAN_ENV).as_deref() == Ok("true");
```

Comment:
this will still be executed on every request because we create new router services on the fly now. The content of the env var should go in a `OnceCell` instead